pkg/decision/entities.go

```diff
@@ -63,3 +63,14 @@ type ExperimentDecision struct {
 	Decision
 	Variation *entities.Variation
 }
+
+// UserDecisionKey is used to access the saved decisions in a user profile
+type UserDecisionKey struct {
+	ExperimentID string
+}
```

Comment:
Why use a struct here, as opposed to just a `string`?

Reply:
Good catch. In our current user profile service implementation we use a combination of experimentID and the key `variation_id`. I don't remember why we decided to make that value a map instead of pointing from experimentID to variationID and I added the struct so we could maintain that relationship and maintain consistency with the other SDKs.